### DISCUSSION TITLE
Folder with period in the name

### DISCUSSION THREAD

**Discussion by bittar** [iterate-ch/cyberduck#14628] [Q&A] [answered]
Hi, it might be a silly question - I'm trying to verify an SSL certificate and need to create a folder named ".well-known"; period included.

Cyberduck won't let me, and I'm not sure if there's another way of doing that.
Any help is appreciated, thanks!

**Comment by bittar** [comment]
Ok, for my future dumbass self: I didn't know periods meant hidden files; problem solved.

**Comment by dkocher** [comment, marked as answer]
Documentation in [Browser → General → Show Hidden Files](https://docs.cyberduck.io/cyberduck/browser/#browser-general-show-hidden-files).
